e2e-tests/demand-backup-cloud/run

```diff
@@ -68,6 +68,58 @@ run_recovery_from_source() {
 	compare_mysql_cmd "select-1" "SELECT * from myApp.myApp;" "-h $cluster-pxc-2.$cluster-pxc -uroot -proot_password"
 }
 
+# If backup upload was started and failed for some reason the cloud storage should be cleaned up during second try
+delete_backup_pod() {
+	local backup_name=$1
+
+	desc "Delete ${backup_name} pod during SST"
+	echo "Waiting for ${backup_name} pod to become Running"
+	sleep 1
+	kubectl wait --for=jsonpath='{.status.phase}'=Running pod --selector=percona.com/backup-job-name=xb-${backup_name} --timeout=120s
+
+	backup_pod=$(kubectl get pods --selector=percona.com/backup-job-name=xb-${backup_name} -o jsonpath='{.items[].metadata.name}')
+
+	echo "Deleting pod/${backup_pod} during SST upload"
+	kubectl logs -f ${backup_pod} | while IFS= read -r line; do
```

Comment:
```suggestion
	kubectl_bin logs -f ${backup_pod} | while IFS= read -r line; do
```

Reply:
Hi, if I use `kubectl_bin logs` here the start of SST is detected too late and pod deleted after the backup finishes.